### PR TITLE
chore: rebrand OpenDIF references to OpenNDX across docs and configs

### PR DESCRIPTION
**Pull request by mushrafmim**
## Summary
Rebrands user-facing references from **OpenDIF** to **OpenNDX** across the repository's documentation, issue templates, and scripts. No functional/code changes — text and labels only.

## Type of Change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update
- [ ] Refactoring (no functional changes)
- [ ] Performance improvement
- [ ] Other (please describe):

## Changes Made
- `README.md` — title, badge, section headings, footer
- `ARCHITECTURE.md` — title, overview, architecture layers, NDX section
- `BUSINESS_WORKFLOW.md` — title and intro
- `SETUP.md` — title and intro
- `init.sh` — startup banner
- `members/die/applications/online-passport-app/README.md` — contributing section
- `.github/ISSUE_TEMPLATE/*.yml` — repo reference updated from `LSFLK/OpenDIF` to `openndx/openndx-farajaland`

## Testing
- [x] I have tested this change locally
- [ ] I have added unit tests for new functionality
- [ ] I have tested edge cases
- [ ] All existing tests pass

## Checklist
- [x] My code follows the project's style guidelines
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have checked that there are no merge conflicts

## Related Issues
Closes #47

## Additional Notes
- The underlying framework's full name — **Open Data Interchange Framework (OpenDIF)** — is intentionally left unchanged; only the product/implementation branding moves to OpenNDX.
- The OpenNDX badge renders as a plain image with no link, since there is no website yet.

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

File: BUSINESS_WORKFLOW.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# OpenDIF Farajaland - Business Workflow
+# OpenNDX Farajaland – Business Workflow
```
[the remark this answers is not in the document]
Comment (mushrafmim) [reply BUSINESS_WORKFLOW.md:1]: Changed `SETUP.md` and `ARCHITECTURE.md` to match this format.

File: ARCHITECTURE.md [hunk 2]
```diff
@@ -255,7 +255,7 @@ The NDX is the core infrastructure layer providing orchestration, consent manage
 
 ### The Challenge
 
-OpenDIF's NDX communicates with data providers using **GraphQL** for all egress calls. However, many existing government systems use legacy protocols like:
+OpenNDX communicates with data providers using **GraphQL** for all egress calls. However, many existing government systems use legacy protocols like:
```
[the remark this answers is not in the document]
Comment (mushrafmim) [reply ARCHITECTURE.md:258]: No need.

File: README.md [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](LICENSE)
-[![OpenDIF](https://img.shields.io/badge/OpenDIF-Reference%20Implementation-green.svg)](https://opendif.org)
+![OpenNDX](https://img.shields.io/badge/OpenNDX-Reference%20Implementation-green.svg)
 
 The **National Data Exchange (NDX)** is a reference implementation of the **Open Data Interchange Framework (OpenDIF)** for the fictional country of Farajaland. NDX demonstrates secure, privacy-preserving data exchange across government agencies through a citizen-centric, consent-based approach.
 
 ## Table of Contents
 
-- [Understanding the OpenDIF Ecosystem](#understanding-the-farajaland-ndx-ecosystem)
+- [Understanding the OpenNDX Ecosystem](#understanding-the-farajaland-ndx-ecosystem)
```
[the remark this answers is not in the document]
Comment (mushrafmim) [reply README.md:10]: Done.

**Review by ginaxu1 — APPROVED** [review]
Lgtm
